### ISSUE TITLE
Oauth2 removed

### ISSUE THREAD

**Issue by evermax** [adlnet/ADL_LRS#400]
Hi,

I am curious to know why you removed oauth2 from the LRS.
Is there a reason for it?

**Comment by ljwolford** [comment]
We added it to give users another auth option but we didn't get any feedback about it. When we checked the public LRS and saw it wasn't even being used (and since it isn't listed in the spec) we just removed it to try to cut down the size of the code base.

**Comment by evermax** [comment]
I see, too bad I used it in a school project.
I am not sure it matters that much because the project will probably never be used again, it was more out of curiosity.

----- Mail original -----
De: "Lou Wolford" <notifications@github.com>
À: "adlnet/ADL_LRS" <ADL_LRS@noreply.github.com>
Cc: "Maxime Lasserre" <maxlasserre@free.fr>, "Author" <author@noreply.github.com>
Envoyé: Jeudi 9 Juin 2016 04:28:13
Objet: Re: [adlnet/ADL_LRS] Oauth2 removed (#400)



We added it to give users another auth option but we didn't get any feedback about it. When we checked the public LRS and saw it wasn't even being used (and since it isn't listed in the spec) we just removed it to try to cut down the size of the code base.
